root/report/ShouldNotHaveDiscIds.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import ReleaseList from './components/ReleaseList';
+import ReportLayout from './components/ReportLayout';
+import type {ReportDataT, ReportReleaseT} from './types';
+
+const ShouldNotHaveDiscIds = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportReleaseT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report shows releases that have at least one medium with a
+       format that does not support disc IDs, yet have disc IDs attached.
+       Usually this means the disc IDs ended up here because of a bug
+       and should be moved or removed.`,
```

Comment:
Couldn't it also mean that the format is wrong?

Reply:
I guess that's less likely given the current ways the bug can happen, but. Hopefully we can fix these + add a database constraint and remove this report one day.

Reply:
It could mean that the format is wrong, yes - something was always meant to be a CD, but got merged into a vinyl by mistake? But I think "usually" is still fair :)